src/wasm-lib/kcl/src/parser/snapshots/kcl_lib__parser__parser_impl__snapshot_math_tests__d.snap

```diff
@@ -1,5 +1,6 @@
 ---
 source: kcl/src/parser/parser_impl.rs
+assertion_line: 3612
```

Comment:
Strange that this field gets added. It's not added when I run the tests. What version of `cargo insta` do you have installed? Were you using `cargo insta` to run these tests?

Reply:
I just run cargo nextest and then move all the new files over the old ones